Branch-TestBed/Branch-SDK-Tests/BranchLoggerTests.m

```diff
@@ -0,0 +1,110 @@
+//
+//  BranchLoggerTests.m
+//  Branch-SDK-Tests
+//
+//  Created by Nipun Singh on 2/5/24.
+//  Copyright © 2024 Branch, Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "BranchLogger.h"
+#import "Branch.h"
+
+@interface BranchLoggerTests : XCTestCase
+@end
+
+@implementation BranchLoggerTests
+
+- (void)setUp {
```

Comment:
Use a local copy instead of the singleton when doing unit tests. Just less likely to run into issues.